rke/k8s_version_info.go

```diff
@@ -606,6 +606,10 @@ func loadK8sVersionInfo() map[string]v3.K8sVersionInfo {
 			MinRancherVersion: "2.6.4-patch0",
 			MinRKEVersion:     "1.3.8-rc0",
 		},
+		"v1.24": {
+			MinRancherVersion: "2.6.6-patch0",
```

Comment:
Need @kinarashah to verify minRancher and minRKE versions, is this correct?

Reply:
the version is updated to `2.6.6-rc0` because there is no patch version for 2.6.6 AFAIK.